nautobot/utilities/tables.py

```diff
@@ -382,3 +404,84 @@ def render(self, record, bound_column, value):
             template = escape(value)
 
         return mark_safe(template)
+
+
+class RelationshipColumn(tables.Column):
+    """
+    Display relationship association instances in the appropriate format.
+    """
+
+    def __init__(self, relationship, side, *args, **kwargs):
+        self.relationship = relationship
+        if relationship.symmetric:
+            self.side = RelationshipSideChoices.SIDE_PEER
+        else:
+            self.side = RelationshipSideChoices.OPPOSITE[side]
+        kwargs["verbose_name"] = relationship.name + " " + self.side
```

Comment:
Should be able to use `relationship.get_label()` here to construct a more self-consistent verbose name.